content/blog/elsewhere/sass-color-draft.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Sass Color Spaces (Request for Comments)
+sub: A new proposal for color-management in Sass
+venue: Sass Blog
+url: https://sass-lang.com/blog/request-for-comments-color-spaces
+date: 2022-09-22
+author: miriam
+image:
+  src: blog/2022/sass-color.jpg
+  alt: |
+    Sass logo in black
+    on top of bright oklch color gradient
+tags:
+  - CSS
+  - Sass
+action:
+  text: Read the summary & proposal »
+  url: https://sass-lang.com/blog/request-for-comments-color-spaces
+summary: |
+  There’s been a lot of exciting work
+  in the CSS color specifications lately,
+  and the new features are starting to land in browsers --
+  so we’ve been preparing to add support in Sass as well.
+  My proposal for that
+  is published and ready for public feedback!
+---
+
+I'm very excited
+about the new color features
+starting to roll out in CSS --
+and similarly excited to make them available
+(with some extra tooling)
+in Sass as well.
+This will allow us to:
+
+- Define colors in new color spaces like `oklch` & `display-p3`
+- Convert colors between spaces
+- Mix & manipulate colors in any space
+- Gamut-map colors for a target space as needed
+
+I hope to write more about color spaces here
+at some point,
+but for now
+check out the [request for comments][request],
+or dive straight into the [full proposal][].
+If you want to learn more about color spaces in CSS,
+Bramus has a great [collection of resources][],
```

Comment:
"Bram.us" the site or "Bramus Van Damme" the person or "Bramus" the handle 🙂? Small thing, but I guess it stuck out sitting next to the other two full names in the sentence.